### PR TITLE
Add 4 new triggers to the connector

### PR DESCRIPTION
**Pull request by RjImpx**
---
When submitting a connector, please check the following conditions for your PR.

- [x] I attest that the connector works and I verified by deploying and testing all the operations.
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a logo.

### CODE REVIEW

File: certified-connectors/Impexium/apiDefinition.swagger.json [hunk 1]
```diff
@@ -2832,7 +2724,7 @@
                   "description": "secret",
                   "title": "",
                   "x-ms-visibility": "internal",
-                  "default": "MSAutomateProdPur"
+                  "default": "MSAutomate"
```
Comment (sriyen-msft) [inline certified-connectors/Impexium/apiDefinition.swagger.json:2727]: @RjImpx I am not sure whether you are aware of it or not, there is no word like `MS Automate` in our product. It was renamed as `Power Automate` long back. If this value appears to public, it need to be changed to `PowerAutomate`.

File: certified-connectors/Impexium/apiDefinition.swagger.json [hunk 2]
```diff
@@ -2861,27 +2753,111 @@
...
+                  "x-ms-notification-url": true,
+                  "x-ms-visibility": "internal",
+                  "title": ""
+                },
+                "description": {
+                  "type": "string",
+                  "description": "description",
+                  "title": "",
+                  "x-ms-visibility": "internal",
+                  "default": "Added By MS Power Automate"
```
Comment (sriyen-msft) [inline certified-connectors/Impexium/apiDefinition.swagger.json:2826]: @RjImpx Same here

File: certified-connectors/Impexium/apiDefinition.swagger.json [hunk 3]
```diff
@@ -2893,77 +2869,342 @@
...
+                  "title": "",
+                  "x-ms-visibility": "internal",
+                  "x-ms-notification-url": true
+                },
+                "description": {
+                  "type": "string",
+                  "description": "description",
+                  "title": "",
+                  "x-ms-visibility": "internal",
+                  "default": "Added By MS Power Automate"
```
Comment (sriyen-msft) [inline certified-connectors/Impexium/apiDefinition.swagger.json:2918]: @RjImpx Same here and everywhere else for similar words.

File: certified-connectors/Impexium/apiDefinition.swagger.json [hunk 4]
```diff
@@ -2893,77 +2869,342 @@
...
+        },
+        "middleName": {
+          "type": "string",
+          "title": "Middle Name",
+          "description": "MiddleName"
+        },
+        "preferredFirstName": {
+          "type": "string",
+          "title": "Preferred First Name",
+          "description": "PreferredFirstName"
```
Comment (sriyen-msft) [inline certified-connectors/Impexium/apiDefinition.swagger.json:3184]: @RjImpx The description values should be free from grammatical and spell mistakes. All similar the description values need to be fixed as they are visible to public.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: certified-connectors/Impexium/apiDefinition.swagger.json (hunk 1 above)
Comment (RjImpx) [reply certified-connectors/Impexium/apiDefinition.swagger.json:2727]: @sriyen-msft we will fix this, I have sent you my comments on isvstudio, can we connect?

Comment (sriyen-msft) [reply certified-connectors/Impexium/apiDefinition.swagger.json:2727]: > @sriyen-msft we will fix this, I have sent you my comments on isvstudio, can we connect?

I have replied to your question in the ISV Studio

File: certified-connectors/Impexium/apiDefinition.swagger.json [hunk 5]
```diff
@@ -2664,117 +2664,186 @@
...
+                  "type": "string",
+                  "description": "Callback URL.",
+                  "x-ms-notification-url": true,
+                  "x-ms-visibility": "internal",
+                  "title": ""
+                },
+                "description": {
+                  "type": "string",
+                  "description": "Description.",
+                  "title": "",
```
Comment (sriyen-msft) [inline certified-connectors/Impexium/apiDefinition.swagger.json:2824]: @RjImpx Can you please let us know why title has empty string.

File: certified-connectors/Impexium/apiDefinition.swagger.json [hunk 6]
```diff
@@ -2893,261 +2960,435 @@
...
             "schema": {
               "type": "object",
               "properties": {
-                "name": {
+                "typeName": {
                   "type": "string",
-                  "description": "Custom Field Name",
-                  "title": "Custom Field Name"
+                  "description": "Type Name.",
+                  "title": "",
```
Comment (sriyen-msft) [inline certified-connectors/Impexium/apiDefinition.swagger.json:2970]: @RjImpx There are many title values are empty, can you please provide valid value for to them. if you have specific reason to keep empty strings, Please let us know

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: certified-connectors/Impexium/apiDefinition.swagger.json (hunk 6 above)
Comment (RjImpx) [reply certified-connectors/Impexium/apiDefinition.swagger.json]: @sriyen-msft These were not visible to the user however we went ahead and updated the titles. thanks

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**